src/components/textbox/textbox.pw.tsx

```diff
@@ -0,0 +1,1244 @@
+              .length;
+            return returnedElement;
+          },
+          { textboxParentParent, firstElement }
+        );
+
+        const domElement2 = await page.evaluate(
+          ({
+            textboxParentParent: parentDomElement,
+            secondElement: element,
+          }) => {
+            const returnedElement = parentDomElement?.querySelectorAll(element)
+              .length;
+            return returnedElement;
+          },
+          { textboxParentParent, secondElement }
+        );
+
+        expect(domElement1).toBeTruthy();
+        expect(domElement2).toBeTruthy();
+      });
+    }
+  );
+
+  /* This test could be deleted, as its specific test case cannot be properly replicated using Playwright,
+  we can assert it exists in the DOM in the specified DOM structure, but cannot properly test its visibility
+  without added complexity or adding new data tags to `Textbox`s DOM. */
+
+  test("should render with positionedChildren prop", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<TextboxComponentWithPositionedChildren />);
+
+    const textboxParent = await textbox(page).evaluateHandle(
+      (element: Element) => {
+        return element.parentElement;
+      }
+    );
+
+    const buttonExistsInTextboxParent = await page.evaluate(
+      (parentDomElement) => {
+        const element = parentDomElement?.querySelectorAll(
+          `[data-component="button"]`
+        ).length;
+        return element;
+      },
+      textboxParent
+    );
+
+    expect(buttonExistsInTextboxParent).toBeTruthy();
+  });
+
+  ([
+    ["flex", 399],
+    ["flex", 400],
+    ["block", 401],
+  ] as [string, TextboxProps["adaptiveLabelBreakpoint"]][]).forEach(
+    ([displayValues, breakpoint]) => {
+      test(`should render with ${displayValues} label alignment when the adaptiveLabelBreakpoint prop is ${breakpoint} with a set viewport of 400`, async ({
+        mount,
+        page,
+      }) => {
+        await page.setViewportSize({ width: 400, height: 300 });
+        await mount(<TextboxComponent adaptiveLabelBreakpoint={breakpoint} />);
+
+        const label = getDataElementByValue(page, "label");
+        const labelParentParent = await label.evaluateHandle(
+          (element: Element) => {
+            return element.parentElement?.parentElement;
+          }
+        );
+
+        const displayValue = await getStyle(
+          (labelParentParent as unknown) as Locator,
+          "display"
+        );
+
+        expect(displayValue).toEqual(displayValue);
+      });
+    }
+  );
+
+  test("should render with labelId prop", async ({ mount, page }) => {
+    await mount(<TextboxComponent id={CHARACTERS.STANDARD} />);
+
+    const label = getDataElementByValue(page, "label");
+
+    await expect(label).toHaveAttribute("id", `${CHARACTERS.STANDARD}-label`);
+    await expect(label).toHaveAttribute("for", `${CHARACTERS.STANDARD}`);
```

Comment:
```suggestion
    await expect(label).toHaveAttribute("for", CHARACTERS.STANDARD);
```

**suggestion** (non-blocking): really minor thing and I know it was like this in the Cypress test - but there is zero value in wrapping a strong in backticks and `${...}`.